topics/ecology/tutorials/openrefine_gbif/tutorial.md

```diff
@@ -92,6 +93,11 @@ This tutorial is based on the GBIF {% cite GBIFtraining2021 %} "Biodiversity dat
 >
 > 1. {% tool [OpenRefine tool](interactive_tool_openrefine) %} with the following parameters:
 >    - *"Input file in tabular format"*: `UC1-3c-open-refine-tabular`
+>    - click the button "run tool"
+> You will then be redirected. A notification "Interactive Tools" shows up on the left-hand side, indicating you have an interactive tool running.
+> Click on the notification, click on the empty box in front of the text saying "Openrfine".
+> Then, click on the text "Openrefine" to open the tool in a new tab.
+>
 > 2. Open the OpenRefine instance
```

Comment:
so, just a quirk of the GTN, but you need to indent these lines to match the bullet point above it, or else the numbering will start at 1 again below (even though in the file it says number 2)

```suggestion
>    - click the button "run tool"
>
>      You will then be redirected. A notification "Interactive Tools" shows up on the left-hand side, indicating you have an interactive tool running.
>      Click on the notification, click on the empty box in front of the text saying "Openrfine".
>      Then, click on the text "Openrefine" to open the tool in a new tab.
>
> 2. Open the OpenRefine instance
```